samples/nrf9160/at_client/prj.conf

```diff
@@ -15,5 +15,5 @@ CONFIG_AT_HOST_LIBRARY=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 
 # Stacks and heaps
-CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_MAIN_STACK_SIZE=2024
```

Comment:
Is this change intended? If so, 2048 perhaps, looks nicer :)

Reply:
Yes, it is intended. The old value was more or less copy&paste from another project. So when Christopher started to query about thread stack size I did some testing with other stack sizes too. This value seems to be stable for this project. But yes, I cange it to 2048 :)